src/mocks/handlers/userHandlers.ts

```diff
@@ -74,6 +80,37 @@ const userHandlers = [
       }
     }
   ),
+
+  graphql.mutation<AddUserMutation, AddUserMutationVariables>(
+    "AddUser",
+    (req, res, ctx) => {
+      const addUser = userFactory.build({
+        username: req.variables.username,
+      });
+      userRepository.save(addUser);
+      return res(ctx.data({ addUser }));
+    }
+  ),
+
+  graphql.mutation<SetUserRoleMutation, SetUserRoleMutationVariables>(
+    "SetUserRole",
+    (req, res, ctx) => {
+      const user = userRepository.find("username", req.variables.username);
+      if (user) {
+        user.role = req.variables.role;
+        userRepository.save(user);
+        return res(ctx.data({ setUserRole: user }));
+      } else {
+        return res(
+          ctx.errors([
+            {
+              message: `Could not find equipment: "${req.variables.username}"`,
```

Comment:
`Could not find user` would make more sense

Reply:
Sorry, typo :) I will correct it.